src/explore-education-statistics-frontend/src/components/PageMeta.tsx

```diff
@@ -30,6 +30,12 @@ const PageMeta = ({
         content="jWf4Mg_pzTOgXDWccGcv9stMsdyptYwHeVpODHdesoY"
       />
 
+      {process.env.APP_ENV !== 'Production' && (
+        <>
+          <meta name="robots" content="noindex,nofollow" />
+          <meta name="googlebot" content="noindex,nofollow" />
```

Comment:
Pretty sure we don't need this as the `robots` tag should be sufficient to block all crawlers, including Google's according to [their docs](https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag): 

> In this example, the robots meta tag instructs search engines not to show the page in search results. The value of the name attribute (robots) specifies that the rule applies to all crawlers.

Reply:
I just erred on the side of caution and didn't see a downside to supplying both, but fine can remove